api/db/changelog/db.changelog-58-update-access-modules.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="ustutsman" id="db.changelog-58-update-access-modules">
+    <addColumn tableName="user">
+      <column name="data_use_agreement_completion_time" type="datetime"/>
+      <column name="data_use_agreement_bypass_time" type="datetime"/>
+      <column name="compliance_training_expiration_time" type="datetime"/>
```

Comment:
I know the ticket wasn't very precise about how to handle column renames like this... sorry about that!

After looking through the PR I think for this particular field (compliance_training_expiration_time), we should exclude it from this PR and handle the column rename (training_expiration_time --> compliance_training_expiration_time) as a separate follow-up. It'll help keep this PR simpler, and won't affect the introduction of the core access module fields.